.github/pull_request_template.md

```diff
@@ -33,14 +33,17 @@ If you need to provide more details, please use the description section.
 -->
 
 - [ ] All files present ( component - md - scss - spec - stories )
-- [ ] Are all props ok and documented
-- [ ] Required props and default values
+- [ ] All props ok and documented (propTypes)
+- [ ] Required props, default values, variants and colors
 - [ ] Example in storybook
+- [ ] Reversed style (light/dark mode)
+- [ ] Allows custom props
 
 ## Documentation
 
 <!--
 Indicate if you have been writing documentation has part of this change.
 -->
 
-- [ ] README documentation
+- [ ] README documentation (Where/when and how the component should be used, structure, responsiveness, dependencies on other components, states, behaviours, animations, accessibility)
```

Comment:
The where/when and how the component should be used is a part of the figma component documentation right now as it's coming from Design.

Also, what is meant by "structure" here?

Reply:
OK with you where/when statement.

Looking at https://docs.google.com/document/d/1moEAolz106XH3XWeYH4-C6Zse2MfAVa5_ywZON_CjY8/edit, I think the idea behind structure is basic specification of the component

Reply:
Ok 👍🏾